src/ConnectedMode/ConnectedMode.csproj

```diff
@@ -26,6 +26,14 @@
 
   </PropertyGroup>
 
+  <ItemGroup>
+    <None Remove="Embedded\SonarLintTargets.txt" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <EmbeddedResource Include="Embedded\SonarLintTargets.txt" />
+  </ItemGroup>
+
```

Comment:
Minor: I would merge these into the same `ItemGroup` to make it more obvious they are related.

Also, I think that adding a `Label` to the `ItemGroup` will stop Visual Studio from adding/removing unrelated entries from it when it is making changes to the project file behind the scenes e.g.

```
  <ItemGroup Label="Global targets file">
    <None Remove="Embedded\SonarLintTargets.txt" />
    <EmbeddedResource Include="Embedded\SonarLintTargets.txt" />
  </ItemGroup>
```